examples/07-advanced-modeling.ipynb

```diff
@@ -94,11 +94,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "ON_GPU = True  # Set `ON_GPU=True` if cuda-enabled GPU is availabe,\n",
+    "ON_GPU = True  # Set `ON_GPU` ti `True` if cuda-enabled GPU is available,\n",
```

Comment:
`ON_GPU` to `True`?

Reply:
Since the notebook crashes without GPU, I rewrote this saying that ON_GPU = False is not allowed. Some other notebooks have been written so that they still run successfully, though slowly, without GPU. But that is not the case here.